src/helpers/terminal_helpers.sh

```diff
@@ -3,6 +3,19 @@
 # Helper methods for writing to the terminal.
 
 
+# http://stackoverflow.com/a/12202793/3418047
```

Comment:
The externally linked site might go away in the future. Please also provide a summary of the comment here, and use the URL only as a link to read more about this issue, or as proof for your claims here.